cdap-ui/app/cdap/components/PipelineList/DeployedPipelineView/PipelineTable/index.tsx

```diff
@@ -76,19 +76,20 @@ const PipelineTableView: React.SFC<IProps> = ({
   return (
     <div className="grid-wrapper pipeline-list-table">
       <div className="grid grid-container">
-        <div className="grid-header">
-          <div className="grid-row">
-            <SortableHeader columnName="name" />
-            <strong>{T.translate(`${PREFIX}.type`)}</strong>
-            <strong>{T.translate(`${PREFIX}.status`)}</strong>
-            <strong>{T.translate(`${PREFIX}.lastStartTime`)}</strong>
-            <strong>{T.translate(`${PREFIX}.nextRun`)}</strong>
-            <strong>{T.translate(`${PREFIX}.runs`)}</strong>
-            <strong>{T.translate(`${PREFIX}.tags`)}</strong>
-            <strong />
+        {pipelines && pipelines.length > 0 && filteredList && filteredList.length > 0 && (
```

Comment:
we have an `If` component that will conditionally render based on some condition. Would be good to use that `components/If`.

Also is the pipelines.length check necessary? Would checking for filteredList alone be enough?

Reply:
Made changes according to the comments!